src/Nethermind/Nethermind.Core/ConsensusRequests/ConsensusRequest.cs

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Nethermind.Core.ConsensusRequests;
+
+public enum ConsensusRequestsType : byte
+{
+    Deposit = 0,
+    WithdrawalRequest = 1
+}
+
+public class ConsensusRequest
```

Comment:
Maybe out of scope of this PR and we can do that in the next one: Shouldn't we create proper class hierarchy deserialization?

Reply:
Not put every property in base class, but put them in correct derived classes:

https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/polymorphism

Reply:
Even better: https://stackoverflow.com/a/59744873/1187056